templates/default/src/__tests__/pages/__snapshots__/_document.test.jsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`_document runs without error 1`] = `<DocumentFragment />`;
```

Comment:
I know this isn't a great test, but for now there isn't a convenient way to test _document.js in next that i've been able to find. I've spent the most hours just researching this. AT the very least we now have 100% code coverage of it and know it doesn't break.

Reply:
I did similar previously, I'm not too concerned with this approach. It probably will make sense longer-term to mock the children of document so that we aren't updating the snapshot on every change of a child component.